src/hooks/core/chat/useMessageHook.ts

```diff
@@ -5,8 +5,9 @@ import ChatSchema from '../../../database/schema/ChatSchema';
 import ShareUtils from '../../../utils/share';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import {Anonimity} from '../../../../types/component/AnonymousChat/BaseChannelItem.types';
 
-const useMessageHook = (message: ChatSchema | undefined) => {
+const useMessageHook = (message: ChatSchema | undefined, anonimity: Anonimity) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `Anonimity` type and its use as a parameter in the `useMessageHook` function is a good change. It makes the function more flexible and adaptable to different chat types. However, it would be better if you could provide a default value for the `anonimity` parameter to ensure backward compatibility with existing calls to this function that do not provide this argument.
```diff
- const useMessageHook = (message: ChatSchema | undefined, anonimity: Anonimity) => {
+ const useMessageHook = (message: ChatSchema | undefined, anonimity: Anonimity = 'SIGNED') => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->